pyproject.toml

```diff
@@ -1,13 +1,12 @@
-[tool.pytest.ini_options]
-addopts = [
-    "--color=yes",
-    "--import-mode=importlib",
-    "--verbose",
-]
-testpaths = [
-    "tests",
+[tool.mypy]
+exclude = [
+    'tests\/data\/test_package_generation\/src\/cookiecutter_test\/__init__\.py',
 ]
 
+[tool.pytest.ini_options]
+addopts = ["--color=yes", "--import-mode=importlib", "--verbose"]
+testpaths = ["tests"]
```

Comment:
This should be in a separate PR

Reply:
I see what you've done now. This change shouldn't be done here. It's not in keeping with the rest of the file

Reply:
Sorry, I had an overzealous linter. Should be fixed now